pkg/database/mysql.go

```diff
@@ -0,0 +1,26 @@
+package database
+
+
+type MySQLDatabase struct {
+	DB *bun.DB
+}
+
+var (
+	once sync.Once
+	dbInstance *MySQLDatabase
+)
+
+func NewMySQLDatabase(conf *entities.Config) *MySQLDatabase {
+	once.Do(func() {
+		sqlDB, err := sql.Open("mysql", "")
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/8#issuecomment-2416672002>#7006ea</a><span> - 10/16/2024, 06:02 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The MySQL connection string is empty, which will prevent successful database connection. The connection parameters should be properly configured using the provided configuration.
</div>
<div id="fix">
<b>Fix:</b> Construct the MySQL connection string using the configuration parameters provided in the conf object.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -15,5 +15,10 @@
  func NewMySQLDatabase(conf *entities.Config) *MySQLDatabase {
  	once.Do(func() {
 -		sqlDB, err := sql.Open("mysql", "")
 +		connStr := fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?parseTime=true",
 +			conf.DBUser,
 +			conf.DBPassword,
 +			conf.DBHost,
 +			conf.DBPort,
 +			conf.DBName)
 +		sqlDB, err := sql.Open("mysql", connStr)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged